nessie/externals/boac.py

```diff
@@ -31,11 +31,14 @@
 
 
 def kickoff_refresh():
-    return authorized_request(app.config['BOAC_CACHE_REFRESH_URL'])
+    successful = True
+    for boac in app.config['BOAC_REFRESHERS']:
+        successful = authorized_request(boac['URL'], boac['API_KEY']) and successful
```

Comment:
@raydavis Should we log a warning if request fails?

Reply:
Thanks, @johncrossman . I thought underlying code took care of that, but I'll double check.

Reply:
Confirmed, @johncrossman -- a failing response is logged at ERROR.